tox.ini

```diff
@@ -11,11 +11,12 @@ deps =
     test: pytest-cov
     test: pytest-mpi>=0.2
 
+    # For --pre, 2.0.0b1 has a header size difference that has been reverted, so avoid it
     py38-deps: numpy>=1.17.5
-    py39-deps: numpy>=1.19.3
-    py310-deps: numpy>=1.21.3
-    py311-deps: numpy>=1.23.2
-    py312-deps: numpy>=1.26.0
+    py39-deps: numpy>=1.19.3,!=2.0.0b1
+    py310-deps: numpy>=1.21.3,!=2.0.0b1
+    py311-deps: numpy>=1.23.2,!=2.0.0b1
+    py312-deps: numpy>=1.26.0,!=2.0.0b1
```

Comment:
Fixes failures in this PR and in `master` like

https://dev.azure.com/h5pyappveyor/h5py/_build/results?buildId=1914&view=logs&jobId=a5aa7219-b716-5eaa-2dae-b4098305f378